test/describe-test-name/test/runner.test.ts

```diff
@@ -0,0 +1,14 @@
+import { expect, test } from 'vitest'
+import { resolve } from 'pathe'
+import { runVitest } from '../../test-utils'
+
+test('should print function name', async () => {
+  const filename = resolve('./fixtures/test/example.test.ts')
+  const { stderr } = await runVitest({ root: './fixtures' }, [filename])
```

Comment:
I am not sure why do tests need to fail? I think we can just add these `describe`/`test` calls to test/reporters 🤔

Reply:
We could also let them pass. Thanks for the suggestion, finding the perfect spot is challenging. 😅